### DISCUSSION TITLE
`appendTo` props for filter Element DataTable

### DISCUSSION THREAD

**Discussion by aemzayn** [primefaces/community#1121] [PrimeReact]
I am using primereact with leaflet map. I can show a DataTable inside a Dialog when leaflet map is on fullscreen with `appendTo` props to Dialog, but because column does not have `appendTo` props for filter element, the filter element stays under the map.

**Comment by melloware** [comment]
What version of PrimeReact? And can you create a StackBlitz reproducer showing the issue?

**Reply by aemzayn** [reply]
I am using the version 7.2.0, but even if I use the newest version I don't see any API of `appendTo` props for Filter Element in Column. I have made an example using React leaflet and Primereact with data table inside a dialog

https://stackblitz.com/edit/stackblitz-starters-q2c4kx?file=src%2FApp.js

As you can see you can open the dialog and filter the datatable normally, but if you set the map to fullscreen you cannot do filter on the datatable because the filter element stays behind the map.

**Reply by melloware** [reply]
I don't think its an AppendTo issue is it isn't it a Zindex issue that the Leaflet Map is now controlling the screen in FullScreen Mode?  Also I know you are using 7.2.0 but i don't think you will get much help on fixing bugs in 7.2.0 when 10.5.0 is the latest version.

**Reply by aemzayn** [reply]
I think leaflet uses a javascript API to do fullscreen, so even if you give very high value for the zIndex it still will not work, as you can see I give the dialog an appendto props so it can render on top of the map. 

Yes I am using version 7.2.0 but atleast you can add this feature in the latest version.

**Reply by melloware** [reply]
So not sure if 7.2.0 has this global `appendTo` config: https://primereact.org/configuration/#appendto

**Reply by aemzayn** [reply]
Version 7.2.0 does not use the primereact provider, but does this global `appendTo` also work for filterElement in column? because that is the main problem.

**Reply by melloware** [reply]
yeah try the static `PrimeReact.appendTo` setting.

**Reply by aemzayn** [reply]
How can I do that?
